qtpy/QtDBus.py

```diff
@@ -4,16 +4,24 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
-"""Provides QtDBus classes and functions."""
 
-# Local imports
+"""
+Provides QtDBus classes and functions.
+"""
+
+import platform
 from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError
 
-if PYQT5:
-    from PyQt5.QtDBus import *
-elif PYQT6:
-    from PyQt6.QtDBus import *
-elif PYSIDE6:
-    from PySide6.QtDBus import *
+if platform.system() == 'Linux':
+    if PYQT5:
+        from PyQt5.QtDBus import *
+    elif PYQT6:
+        from PyQt6.QtDBus import *
+    elif PYSIDE2:
+        raise PythonQtError('QtDBus not implemented in PySide2')
+    elif PYSIDE6:
+        from PySide6.QtDBus import *
+    else:
+        raise PythonQtError("No Qt bindings could be found")
 else:
-    raise PythonQtError('No Qt bindings could be found')
+    raise PythonQtError('QtDBus does not exist on this operating system')
```

Comment:
```suggestion
    raise QtModuleNotFoundError(
        name='QtDBus', msg='{name} does not exist on this operating system.')

```

Since this is, AFAIK, a single special case of a OS-specific Qt module (or is it?), we could use a standard `QtModuleNotFoundError` for this, unless we want to create another error class.

Also, missing EOF EOL

Reply:
There's also `QtWinExtras`, `QtMacExtras`, and `QtX11Extras` that are all platform specific (they also only exist in Qt5, but in both APIs).

I think QtDBus is a special case but only in PySide2 and PySide6. It appears to present in PyQt5 and PyQt6 even on Windows.

I would really like to expand `module_reports` to use the CI system(or something?) to generate data directly on all 3 platforms. This would give us a canonical answer to these edge cases instead of trying to poke-and-hope.